scene/main/resource_preloader.cpp

```diff
@@ -40,10 +40,9 @@ void ResourcePreloader::_set_resources(const Array &p_data) {
 	ERR_FAIL_COND(names.size() != resdata.size());
 
 	for (int i = 0; i < resdata.size(); i++) {
-		String name = names[i];
 		Ref<Resource> resource = resdata[i];
 		ERR_CONTINUE(!resource.is_valid());
-		resources[name] = resource;
+		resources[names[i]] = resource;
 
 		//add_resource(name,resource);
```

Comment:
```suggestion
		//add_resource(names[i],resource);
```
To prevent confusion if this comment is restored